trace_decoder/src/interface.rs

```diff
@@ -162,6 +166,16 @@ pub enum ContractCodeUsage {
     /// contract code will not appear in the [`BlockTrace`] map.
     Write(#[serde(with = "crate::hex")] Vec<u8>),
 }
+// Review: Re-adding this. It has been removed upstream.
```

Comment:
This isn't used in `trace_decoder`.

Make it a free function in `zero`, it's not worth a method here

Reply:
OK.